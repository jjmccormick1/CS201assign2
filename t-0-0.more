s
r
d  D 
d  X 
d "l"
i "L"
d "p"
d "f"
d  o 
d  C 
i  n 
d " "
i  h 
i  G 
d ";"
d "J"
d " "
i  K 
s
r
