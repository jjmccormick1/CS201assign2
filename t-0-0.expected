0: f(f)X
1: d[2]+(f)L k(f)R
2: =c(d[2]+)L =j(k)L =z(k)R
Duplicates: 1
Nodes: 6
Minimum depth: 1
Maximum depth: 2
x not found
l not found
p not found
o not found
0: l+(l+)X
1: g-(l+)L z+(l+)R
2: =d(g-)L k[2]+(g-)R =n(z+)L
3: =h(k[2]+)L
Duplicates: 2
Nodes: 7
Minimum depth: 1
Maximum depth: 3
